libs/styles/src/theme/_default-shadows.scss

```diff
@@ -1,3 +1,12 @@
 $default-shadows: (
```

Comment:
Let's update this to use maybe tailwind values? Also add drop shadow

Reply:
Good idea!

I updated the values to tailwind and also added the drop shadow from tailwind to the config